pages/common/now.md

```diff
@@ -0,0 +1,27 @@
+# now
+
+> Cloud platform for serverless deployment.
+
+- Deploy current directory:
+
+`now`
+
+- List of deployments:
+
+`now list`
+
+- Display information related to a deployment:
+
+`now inspect {{deployment_url}}`
+
+- Remove a deployment:
+
+`now remove {{deployment_id}}`
+
+- Log in into account or create a new one:
+
+`now login`
+
+- Initialize an example project:
```

Comment:
Same thing here. Could this be the 2nd example please? Also, perhaps this would be clearer?

```suggestion
- Initialize an example project in the current directory:
```

Reply:
Actually this not like `npm init` command, I don't know why it is named init (`now example` would be better), but it is not necessary command. But interesting command for reveal now's opportunities, that's why I propose it at the end.

Reply:
If you push the new commit, the review comments should still apply (or will show outdated if you change content marked by a review).

Reply:
Sounds good to me!